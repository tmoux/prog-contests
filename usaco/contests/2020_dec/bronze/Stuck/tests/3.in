20
E 17 29
E 30 12
N 29 2
E 8 19
N 43 6
N 23 21
N 34 37
E 24 15
E 13 26
N 20 46
N 5 25
E 27 36
N 32 45
E 14 43
E 37 8
E 28 38
E 3 1
E 49 32
E 26 18
N 36 44
