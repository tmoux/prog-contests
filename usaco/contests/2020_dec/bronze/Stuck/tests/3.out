26
13
13
35
30
8
Infinity
19
10
Infinity
Infinity
Infinity
Infinity
20
6
6
Infinity
Infinity
17
Infinity
